Dockerfile

```diff
@@ -70,7 +70,7 @@ COPY start.sh /usr/local/bin/start.sh
 RUN chmod +x /usr/local/bin/start.sh
 
 COPY environment.conf.json /environment.conf.json
-COPY environment.conf.schema.json /environment.conf.schema.json
+COPY internals/schemas/environment.conf.schema.json /environment.conf.schema.json
```

Comment:
It needs to be placed in the same location in the container. i.e.

    COPY internals/schemas/environment.conf.schema.json /internals/schemas/environment.conf.schema.json

Otherwise the [validation script](https://github.com/Amsterdam/signals-frontend/pull/1033/files#diff-719e8ac6606473670bd5d07c12443218) won't work on the container

Reply:
Thanks! It was on my list to test, but first wanted to await the reviews, will fix and double check!